conftest.py

```diff
@@ -46,25 +31,12 @@ def is_testrpc_provider(provider):
     return isinstance(provider, EthereumTesterProvider)
```

Comment:
You might be thinking of:
```python
if not isinstance(w3.provider, EthereumTesterProvider):
        w3.provider = EthereumTesterProvider()
```
that was pulled out of `test_existing_filter_instance`? But I'm not sure about removing `skip_if_testrpc`. It would touch a bunch of new files, so I'll try it as a separate PR off master.

Reply:
Removing `skip_if_testrpc` everywhere else does seem to break things, mostly places where `eth_tester` API endpoints aren't implemented. I'll make a note for further discussion.